sound/voice_data.h

```diff
@@ -35,7 +33,10 @@ const uint8_t spGOOD[] PROGMEM = {0x0A,0x28,0xCD,0x34,0x20,0xD9,0x1A,0x45,0x74,0
 const uint8_t spPAUSE1[] PROGMEM = {0x00,0x00,0x00,0x00,0xFF,0x0F};
 const uint8_t spVOLTS[] PROGMEM = {0xe8,0xc8,0xaa,0xd3,0x4d,0x1d,0x23,0x3f,0xa8,0x76,0x51,0x73,0x4c,0x22,0xd4,0x4a,0x3d,0x4c,0xb2,0x89,0x2c,0x4f,0x8f,0x34,0xcb,0x25,0x0c,0xae,0x54,0xd3,0x63,0xb7,0x28,0xc4,0x76,0xee,0x96,0xdd,0xe2,0x98,0xc6,0xa1,0x8b,0x76,0x4b,0x7c,0x5c,0xa3,0x2c,0x29,0x2d,0x0d,0x7e,0x03,0xaa,0x24,0x97,0x2c,0xab,0x6d,0x94,0x31,0x95,0x8a,0x68,0x36,0xc1,0xd6,0xb2,0x2b,0x83,0xde,0x04,0x6b,0xdb,0xaa,0xf4,0x61,0x04,0xba,0x6d,0xb1,0xca,0xaa,0x31,0x71,0x77,0x82,0xca,0x24,0xd2,0xd5,0x54,0x0e,0xfa,0x8b,0xec,0x0c,0x55,0x27,0x08,0x58,0x36,0x82,0x00,0xd3,0x67,0x08,0x60,0xf9,0x0c,0x01,0x6c,0x5b,0xa1,0x80,0xe5,0x2a,0x15,0xb0,0x43,0x96,0x00,0x76,0xc8,0x22,0xc7,0x48,0xd3,0xea,0xea,0xe6,0x01};
-#else
+const uint8_t spAUTO[] PROGMEM = {0xac,0x4c,0x2d,0x8c,0x57,0x9d,0xb8,0x32,0xed,0x34,0x7e,0x8a,0x53,0xca,0x78,0xc2,0x75,0xb5,0x4a,0x51,0xe2,0x8d,0xd0,0xb5,0xb2,0x43,0x49,0xbd,0x4a,0x3a,0x6a,0x2f,0x25,0xd5,0x4c,0xcd,0x8a,0x33,0x94,0x9c,0xbc,0xa4,0x63,0xac,0x33,0x72,0xd2,0xe2,0xf6,0x32,0xc6,0xca,0x96,0x4a,0xc6,0x2a,0x17,0xa7,0x04,0x0e,0xde,0x2c,0xdb,0xbc,0x12,0xa4,0x78,0xb3,0x6c,0x0b,0x4a,0x90,0xe2,0x8f,0x72,0x23,0x2a,0x4e,0x5b,0xa7,0x63,0x8f,0x38,0x3b,0x1d,0xad,0xaa,0x5d,0xe2,0xec,0x75,0x68,0xb3,0x6a,0x88,0x63,0xb2,0xc6,0xf7,0xb2,0x21,0x0e,0xc5,0x13,0x37,0xca,0xaa,0xd8,0x0e,0x73,0x7a,0x0f,0x23,0x62,0xbd,0xd5,0xe8,0x2c,0x16,0x89,0xf5,0x16,0xe5,0xf1,0x84,0x28,0xe6,0xcd,0x15,0x47,0x55,0xa3,0x98,0xc5,0x30,0x0e,0x77,0x84,0xa2,0x20,0x23,0x58,0xcd,0x31,0x3c};
+const uint8_t spKILL[] PROGMEM = {0x0a,0x98,0x49,0x2a,0x00,0xab,0x69,0x15,0xa7,0x18,0x8a,0x98,0x74,0xb3,0x9c,0xea,0x28,0xe5,0x34,0xf6,0x71,0x9b,0xe5,0x94,0x6d,0xc5,0xc3,0x2b,0x56,0x8a,0xdf,0x63,0x37,0x3f,0x07,0x2b,0x3a,0x2b,0x5b,0x82,0x94,0x3d,0xe8,0xb5,0x6c,0x89,0x62,0x8e,0xa0,0xf7,0xb0,0x29,0x8e,0x35,0x8d,0xde,0xcb,0xba,0xd8,0xb7,0x32,0x7a,0x35,0x67,0x12,0xdf,0x4b,0xf1,0xc5,0x9e,0x48,0xfd,0x4c,0xa6,0x27,0xc7,0x2c,0xf5,0x5b,0x49,0x5f,0x18,0x93,0xd4,0x6d,0x26,0x3f,0xa1,0x4d,0x12,0x3d,0x0c,0xb9,0x53,0x16,0x4a,0xac,0x6f,0x47,0x0c,0x57,0x28,0x2e,0xdc,0x42,0xc5,0x68,0xc1,0x03};
+const uint8_t spSTUN[] PROGMEM = {0xe6,0x9f,0xa9,0xdd,0x2b,0xed,0x10,0x60,0x99,0x0e,0x02,0x4c,0xd3,0x41,0x80,0x69,0x3b,0x11,0xb0,0x5d,0x17,0x10,0x60,0xa9,0xcc,0xe6,0x36,0xe1,0xa6,0x1e,0xd6,0x96,0x97,0xad,0x85,0x8e,0x2a,0x3e,0x5e,0x0e,0x11,0xda,0x22,0x7b,0xf9,0x39,0xa5,0x6a,0x29,0xe3,0x11,0xe6,0x1c,0xe2,0x29,0xb2,0x57,0x5c,0x52,0xaa,0x96,0xc8,0x5e,0x49,0x0d,0xa9,0xda,0x2a,0xbb,0x65,0xd5,0xbb,0xca,0xb0,0xec,0x90,0x45,0x23,0x6a,0xad,0xaa,0x4d,0xae,0x3c,0x57,0x85,0x78,0x76,0x85,0x76,0x12,0x95,0xaa,0xc7,0x14,0xdc,0x6b,0x66,0xaa,0x6f,0x55,0xd0,0x60,0x15,0x25,0x9e,0x55,0xc1,0x74,0x64,0x74,0x70,0x25,0x45,0x34,0xaa,0xe6,0xc6,0x16,0x1e};
+#else // Male Voice
```

Comment:
This is confusing, Move the comment after the #else

Reply:
Moved.